themes/V3/5ePHB/snippets.js

```diff
@@ -21,9 +21,41 @@ module.exports = [
 		view      : 'text',
 		snippets  : [
 			{
-				name : 'Table of Contents',
-				icon : 'fas fa-book',
-				gen  : TableOfContentsGen
+				name        : 'Table of Contents',
+				icon        : 'fas fa-book',
+				gen         : TableOfContentsGen,
```

Comment:
add `experimental : true` to the top level item, i think.  For the "Beta" tag.  I don't know if it makes much sense to put it on every sub item in the menu, since it's all very much linked together.